gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -131,12 +131,17 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
 
         return api_endpoint.replace(".googleapis.com", ".mtls.googleapis.com")
 
+    # Note: DEFAULT_ENDPOINT is deprecated. Use DEFAULT_ENDPOINT_TEMPLATE instead.
```

Comment:
Deprecation note for `DEFAULT_ENDPOINT`. This should no longer be used anywhere in the code except for `_get_mtls_endpoint_and_cert_source` tests which will eventually be removed once we clean up the code.